docs/_includes/home-lhn.html

```diff
@@ -1,6 +1,6 @@
 <ul class="lhn-items">
     <li class="home-link">
-        <div class="link" onclick="navigateToHome()">Home</div>
+        <div class="selected">Home</div>
```

Comment:
Is there a reason this doesn't need to navigate anymore?

Reply:
Ah yeah, because this is HTML that corresponds to the LHN on the home page. There are going to be only three HTML files for the LHN (home, send money hub, and request money hub) and we're going to select the option in the LHN according to where the user has navigated to. 

For example, this is how the LHN looks on the home page (where the Home link is disabled):
<img width="289" alt="Screen Shot 2022-08-03 at 14 49 19" src="https://user-images.githubusercontent.com/6829422/182696635-c747cdb7-bb13-4902-a5da-4534845e47f4.png">

And this is going to be for the request money hub:
<img width="292" alt="Screen Shot 2022-08-03 at 14 49 24" src="https://user-images.githubusercontent.com/6829422/182696745-e9ad33c5-97c7-47c5-b569-d43c285c9d3b.png">